decidim-admin/spec/policies/participatory_process_policy_spec.rb

```diff
@@ -38,7 +38,7 @@ module Admin
       end
 
       context "index?" do
-        let(:process) { [create(:process, organization: organization)] }
+        let(:process) { [create(:participatory_process, organization: organization)] }
```

Comment:
Because it's the `index` action. Actually, I should rename `process` to `processes` to make it clearer